.circleci/config.yml

```diff
@@ -30,7 +30,8 @@ anchors:
   env-default: &env-default
     JAVA_TOOL_OPTIONS: -Xmx2g
     # See: https://support.circleci.com/hc/en-us/articles/360021812453-Common-Android-memory-issues
-    GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2 -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy=in-process
+    # https://stackoverflow.com/questions/38967991/why-are-my-gradle-builds-dying-with-exit-code-137
+    GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2 -Dkotlin.incremental=false -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.jvmargs=-XX:+UseContainerSupport
```

Comment:
`UseContainerSupport` should be enabled by default.  Enable in case it isn't.  https://stackoverflow.com/questions/54516988/what-does-usecontainersupport-vm-parameter-do